### DISCUSSION TITLE
init.lua equivalent of `:source $MYVIMRC`?

### DISCUSSION THREAD

**Discussion by mhanberg** [neovim/neovim#13941] [Q&A] [answered]
I've recently switched to `init.lua` and it's not clear to me how to quickly source my configuration without restarting my editor. When i was using init.vim, I had a mapping of `<leader>sv` to source my init.vim and it was very helpful.

Is there currently a solution to this?

**Comment by mjlbach** [comment, marked as answer]
Yep: `vim.api.nvim_set_keymap('n', '<leader>sv', ":luafile ~/.config/nvim/init.lua<CR>", { noremap=true })`

**Reply by mhanberg** [reply]
That works, thanks!
